src/main/java/sincity/model/Spawner.java

```diff
@@ -45,4 +48,18 @@ private RoadPuzzle getRandomSpawnPuzzle(List<RoadPuzzle> spawnPuzzles) {
         int randomIndex = (int) Math.floor(Math.random() * spawnPuzzles.size());
         return spawnPuzzles.get(randomIndex);
     }
+
+    private Direction getArrivalDirection(RoadPuzzle spawnPuzzle) {
+        Direction arrivalDirection = null;
+        if (spawnPuzzle.getIndexX() == 0) {
```

Comment:
Maybe, it would be more readable, if there are fields with booleans like isUpper/Down/Left/RightPadding according to coordinates == 0. ( I mean: boolean isUpperPadding = (x == 0) etc.); I can see we use padding searching by checking if x or y is 0 at least twice...

Reply:
Good idea 👍